holo-vrrp/src/packet.rs

```diff
@@ -463,32 +459,4 @@ impl ArpHdr {
         buf.put_ipv4(&self.target_proto_address);
         buf
     }
-
-    pub fn decode(data: &[u8]) -> DecodeResult<Self> {
```

Comment:
Couldn't we remove the  `decode` routine from `EthernetHdr` as well?

Reply:
You're right. Doing so in a minute.